en/flying/pre_flight_checks.md

```diff
@@ -28,7 +28,7 @@ The following errors (with associated checks and parameters) are reported by the
 `PREFLIGHT FAIL: EKF YAW ERROR`:
 * This error is produced when the yaw angle estimated using gyro data and the yaw angle from the magnetometer or external vision system are inconsistent.
 * Check the IMU data for large yaw rate offsets and check the magnetometer alignment and calibration.
-* The check is controlled by the [COM_ARM_EKF_POS](../advanced_config/parameter_reference.md#COM_ARM_EKF_POS) parameter
+* The check is controlled by the [COM_ARM_EKF_YAW](../advanced_config/parameter_reference.md#COM_ARM_EKF_YAW) parameter
```

Comment:
@NicolasM0 Thanks for this. I'm a little concerned that some of the information from the removed section might be useful for people to know - e.g. the stuff around the default value setting. Thoughts?

@MaEtUgR Can you confirm this is OK, or suggest who might also review?

Reply:
I didn't add information from the removed section to be homogeneous with the other sections. But if you think it can be useful, I will add a point with: 
"The default value of 0.5 allows the differences between the navigation yaw angle and magnetic yaw angle (magnetometer or external vision) to be no more than 50% of the maximum tolerated by the EKF and provides some margin for error increase when flight commences. It can fail if the yaw gyro has a large offset or if the vehicle is moved or rotated in the presence of a bad magnetic interference or magnetometer calibration."

Reply:
Thanks! I'm hoping @MaEtUgR can comment; I'm no expert in this so just attempting to facilitate by bringing in someone who actually knows what they are talking about to to review :-)

Reply:
I honestly don't know what the parameter exactly does, would need to check in the code. @bresch was likely already looking into it recently. I'll ask him.

Reply:
Thanks, looking forward to @bresch commenting!

Reply:
Good catch @NicolasM0 , it's absolutely correct.
Can you add the point with your description of the test ratio?